shownotes/S01E20-开源之夏学生专访-王嘉树.md

```diff
@@ -0,0 +1,53 @@
+---
+num: 'S01E20'
+date: 2023-05-05
+title: '开源之夏学生专访-王嘉树'
+link: ''
+guests:
+- wjsvec
+hosts:
+- linuxsuren
+- MikeyWei
+---
+
+王嘉树，北京邮电大学研二在读，曾参与开源之夏 [TDengine Visual Studio Code Extension](https://github.com/taosdata/vscode-tdengine) 项目并顺利结题，活跃在 TDengine 社区。目前的兴趣在开源项目方面，曾在科大讯飞、电信研究院等实习。
+
+在这次访谈中，我们大致聊到了如下几点：
+
+* 如何接触到的开源
+* 相比于闭源的、企业内部项目而言，开源项目的吸引力在哪里
+* 参与开源之夏的体验分享
+* 在选择开源之夏项目、导师过程中的一些建议
+
+## 嘉宾
+
+[wjsvec](https://github.com/wjsvec)
+
+## 主持人
+
+- [Mikey](https://github.com/MikeyWei) 开源爱好者、一个码字的
+- [Rick](https://github.com/linuxsuren)（开源爱好者、业余开源布道者）
+
+## 时间线
+
+* 02:38 接触开源的机缘是源自企业对开源项目的诉求
+* 08:23 落选 Nebula Graph
```

Comment:
Sorry about this!!! 😭

Reply:
Actually, this is also a valuable experience.